src/errors/runtime/templates.js

```diff
@@ -111,5 +111,6 @@ export default {
     [RUNTIME_ERRORS.uncaughtErrorInReporter]:           'An uncaught error occurred in the "{reporterName}" reporter\'s "{methodName}" method. Error details:\n{originalError}',
     [RUNTIME_ERRORS.roleInitializedWithRelativeUrl]:    'You cannot specify relative login page URLs in the Role constructor. Use an absolute URL.',
     [RUNTIME_ERRORS.typeScriptCompilerLoadingError]:    'Unable to load the TypeScript compiler.\n{originErrorMessage}.',
-    [RUNTIME_ERRORS.cannotCustomizeSpecifiedCompilers]: 'You cannot specify options for the {noncustomizableCompilerList} compiler{suffix}.'
+    [RUNTIME_ERRORS.cannotCustomizeSpecifiedCompilers]: 'You cannot specify options for the {noncustomizableCompilerList} compiler{suffix}.',
+    [RUNTIME_ERRORS.browserConnectionError]:            '{originalErr}\n{openedConnectionsNum} of the total {totalConnectionsNum} browser connections have established.\n{warnings}\n'
```

Comment:
Hmm, could we do something like this one or it would be too hard:
```
Unable to establish one or more browser connections.

Failed connections:
chrome:headless (2 of 2)
chrome:emulation (1 of 2)
alias (number-of-failed of concurrency-level)

Successful connections:
firefox

Diagnostic messages:
...
```